core/domain/platform_parameter_registry.py

```diff
@@ -355,6 +355,13 @@ def _to_platform_parameter_model(
     platform_parameter_domain.DataTypes.BOOL
 )
 
+Registry.create_platform_parameter(
+    ParamName.ENABLE_VOICEOVER_CONTRIBUTION,
+    'Whether the voiceover contribution should be enabled for all users',
+    platform_parameter_domain.DataTypes.BOOL,
+    default=True
```

Comment:
@seanlip please confirm that you're okay with this being immediately enabled before gating code is in place.

Reply:
Hm, this is actually quite an interesting use of platform parameters (see the video/image in the PR description). It's not "wrong", but I think it creates inconsistencies in the codebase because now one platform parameter is editable in the release coordinators view and the others are not. That's going to make things harder to maintain.

@Nik-09 have you considered using a second feature flag for this instead? The enable/disable makes this seem like a reasonable candidate for that, we just have to make sure to set it to True as part of the code deployment.

Reply:
Updated.
Removed platform parameter variable and created feature flag

Reply:
The latest approach seems much cleaner--thanks!